src/general/StringUtils.cs

```diff
@@ -187,4 +187,37 @@ public static string Unescape(this string input)
 
         return result.ToString();
     }
+
+    /// <summary>
+    ///   Convert a value in a string with up to 3 digits.
+    /// </summary>
+    /// <remarks>
+    ///   If greater than 100, it should use F0, otherwise it should use F1.
```

Comment:
Remarks must contain all the text within `para` elements, as per the styleguide, which you should check if you have not already.